lib/data_cycle/update_cycle_alert_generator.rb

```diff
@@ -43,6 +44,17 @@ def generate_alerts
 
     log_message 'Generate de-userfying edits alerts'
     DeUserfyingEditAlertMonitor.create_alerts_for_deuserfying_edits
+
+    generate_alert_for_med_articles
+  end
+
+  def generate_alert_for_med_articles
+    if Features.wiki_ed?
```

Comment:
For consistency with most of the codebase, I'd prefer this guard as `return unless Features.wiki_ed?` rather than using an `if` block.